### PR TITLE
Feature/create dockerfile for backend

### PR DESCRIPTION
**Pull request by DawidStuzynski**

### CODE REVIEW

File: src/main/resources/application-dev.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ spring:
   banner:
     location: classpath:banner.txt
   datasource:
-    url: jdbc:mysql://localhost:3306/trackexpensesapp?useUnicode=true&serverTimezone=UTC
+    url: jdbc:mysql://database:3306/trackexpensesapp?useUnicode=true&serverTimezone=UTC&useSSL=false
```
Comment (purbanski-deftcode) [inline src/main/resources/application-dev.yaml:10]: Should not this be set via environment variable?

Comment (DawidStuzynski) [reply src/main/resources/application-dev.yaml:10]: Originally was in this way so I didn't change it. I also think that through environment variables would be better. I'll fix it

**Review by purbanski-deftcode — APPROVED** [review]
All works fine for me locally, tested using Insomnia / Postman.

**Review by dawciobiel — APPROVED** [review]
Looks good for me

**Pull request merged**